itest/litd_node.go

```diff
@@ -95,6 +107,15 @@ type NodeConfig struct {
 	AdminMacPath   string
 	ReadMacPath    string
 	InvoiceMacPath string
+	FaradayMacPath string
```

Comment:
Re the prior comment, we could then embed _just_ these new fields within a new struct.